app/helpers/has-focus.js

```diff
@@ -0,0 +1,16 @@
+import Ember from 'ember';
+
+export function hasFocus(selector){
+	return {
+    isDescriptor: true,
+    get: function() {
+    	if(selector === 'undefined' || selector === ''){
+    		return false;
+    	}
+
+    	var activeElement = Ember.$(document.activeElement).get(0);
+    	var element = Ember.$(selector).get(0); 
+      return activeElement === element;
+    }
+  };
+}
```

Comment:
Nice implementation of `hasFocus`, maybe I can steal this for `ember-cli-page-object` :stuck_out_tongue_winking_eye: 

Fix indentation please!